src/src/gas/station/hillel/Main.java

```diff
@@ -0,0 +1,22 @@
+package gas.station.hillel;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Main {
+    public static void main(String[] args) {
+        PetrolStation station = new PetrolStation(100);
+        ExecutorService executorService = Executors.newFixedThreadPool(5);
```

Comment:
ExecutorService should be shutdown after work, so close it manually or try using try-with-resources